.gitignore

```diff
@@ -37,3 +37,6 @@ provider/testdata/data/postcard
 **/_debug/
 
 .venv
+
+# Ignore irrelevant files that get generated by VSCode (debug)
+.vscode/
```

Comment:
Suggestion (optional): There is already a vscode section in this .gitignore file on line 15. Consider adding the exclusion up there.